### PR TITLE
Bundles API enhancement 

### PR DESCRIPTION
**Pull request by Aveti79**
Closes: [Issue#125](https://github.com/crowdin/crowdin-api-client-java/issues/125)

### CODE REVIEW

File: src/main/java/com/crowdin/client/bundles/BundlesApi.java [hunk 1]
```diff
@@ -111,4 +111,56 @@ public ResponseList<? extends FileInfo> listBundleFiles(Long projectId, Long bun
...
+     * @param projectId project identifier
+     * @param bundleId bundle identifier
+     * @param exportId export identifier, consists of 36 characters
+     * @return requested bundle export object
+     * @see <ul>
+     * <li><a href="https://developer.crowdin.com/api/v2/#operation/api.projects.bundles.exports.get" target="_blank"><b>API Documentation</b></a></li>
+     * <li><a href="https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.bundles.exports.get" target="_blank"><b>Enterprise API Documentation</b></a></li>
+     * </ul>
+     */
+    ResponseObject<BundleExport> exportBundleStatus(Long projectId, Long bundleId, String exportId) {
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/client/bundles/BundlesApi.java:160]:
```suggestion
    ResponseObject<BundleExport> checkBundleExportStatus(Long projectId, Long bundleId, String exportId) {
```

File: src/main/java/com/crowdin/client/bundles/model/BundleAttributes.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.crowdin.client.bundles.model;
+
+import lombok.Data;
+
+@Data
+public class BundleAttributes {
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/client/bundles/model/BundleAttributes.java:6]: There is a mistake in the API documentation that will be fixed soon

The bundle attributes should contain the `bundleId` field (integer)

```json
"attributes": {
    "bundleId": 2
},
```

**Review by andrii-bodnar — COMMENTED** [review]
@Aveti79 thanks a lot for your contribution! 🚀 

Please check my comments below

File: src/test/java/com/crowdin/client/bundles/BundlesApiTest.java [hunk 3]
```diff
@@ -109,4 +115,31 @@ public void listFileCollectionResourceTest() {
...
+        request.setExportPattern(pattern);
+        request.setLabelIds(Collections.singletonList(0L));
+
+        ResponseObject<BundleExport> response = this.getBundlesApi().exportBundle(projectId, bundleId, request);
+        assertEquals(exportId, response.getData().getIdentifier());
+        assertEquals(2,response.getData().getAttributes().getBundleId());
+    }
+
+    @Test
+    public void exportBundleStatusTest() {
```
Comment (andrii-bodnar) [inline src/test/java/com/crowdin/client/bundles/BundlesApiTest.java:140]: the name of the test and tested method should also be changed according to the previous comment

**Review by andrii-bodnar — APPROVED** [review]
@Aveti79 LGTM, thank you!

**Comment by Aveti79** [comment]
So happy to see it! This is my first contribution to open source, feels great!

**Comment by andrii-bodnar** [comment]
@Aveti79 congratulations, it's a good start in the open-source world 🚀 

Feel free to check the other issues in the current repository or Crowdin organization - [`good first issue` list](https://github.com/search?q=org%3Acrowdin+label%3A%22good+first+issue%22&state=open&type=Issues)

**Pull request merged**